.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+///////////////////⚡///////////////////
+
+# 🚫 AWESOME A PR!
+
+- You can just delete all of this and start your PR text anytime -
+
+Hello there, just a quick pre-warning of some of the Danger rules we on Danger PRs.
+
+We request that every code change to Danger include a CHANGELOG entry, this is 
+so that:
+
+1. Orta doesn't get all the credit for other people's work
+2. People know what changes are between versions
+
+Danger will look for a modification to the `CHANGELOG.md` when there are changes including
```

Comment:
I thought about it, but people see these as un-rendered markdown, so a link doesn't add much here